### PR TITLE
Added hyperparameter guidelines for DELF training

### PR DESCRIPTION
**Pull request by dan-anghel**
# Description

Updated the DELF documentation to include hyperparameter guidelines.

## Type of change

- [x] Documentation update

## Tests

N/A

## Checklist

- [x] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [x] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [x] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [x] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: research/delf/delf/python/training/README.md [hunk 1]
```diff
@@ -179,6 +175,14 @@ python3 train.py \
   --delg_global_features
 ```
 
+### Hyperparameter Guidelines
+
+In order to improve the convergence of the training, the following
```
Comment (andrefaraujo) [inline research/delf/delf/python/training/README.md:180]: just to clarify, here you mean that all other flags for train.py should use the default values, right? If so, maybe briefly mention it here.

**Review by andrefaraujo — APPROVED** [review]
Looks great, thanks Dan!

File: research/delf/delf/python/training/README.md (hunk 1 above)
Comment (dan-anghel) [reply research/delf/delf/python/training/README.md:180]: Yes, that is correct. I added a mention to clarify this. Thanks for picking this up!

**Comment by dan-anghel** [comment]
Thanks for your feedback Andre. I added the mention about the remaining flags.

**Review by andrefaraujo — APPROVED** [review]

**Pull request merged**